packages/design-system/src/styles/components/_Table.scss

```diff
@@ -55,37 +61,60 @@
     word-wrap: normal;
   }
 
-  th,
-  td {
+  tr,
+  caption {
+    display: block;
+  }
+
+  td,
+  th {
     border: 0;
+    display: grid;
+    grid-template-columns: 42% auto;
+    position: relative;
+    z-index: 100;
+
+    // stylelint-disable selector-no-qualifying-type
+    &[data-title]::before {
+      // stylelint-enable selector-no-qualifying-type
+      content: attr(data-title);
+      font-weight: bold;
+      max-width: 40%;
+      width: 40%;
+    }
+
+    > * {
+      grid-column: 2;
+    }
   }
 
   tr {
+    background-color: color('white');
```

Comment:
Is this supposed to be a variable? this isnt compiling into CSS right for me

Reply:
Removed, it wasn't necessary